src/System.ComponentModel.TypeConverter/tests/BindingListTests.cs

```diff
@@ -0,0 +1,699 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information. 
+
+using System;
+using System.ComponentModel;
+using System.ComponentModel.Design.Serialization;
+using System.Collections.Generic;
+using Xunit;
+using System.Reflection;
+
+namespace System.ComponentModel.Tests
+{
+	public class BindingListTest
+	{
+		[Fact]
+		public void BindingListDefaults ()
+		{
+			BindingList<string> l = new BindingList<string> ();
+			IBindingList ibl = (IBindingList)l;
+				
+			Assert.True (l.AllowEdit, "1");
+			Assert.False (l.AllowNew, "2");
+			Assert.True (l.AllowRemove, "3");
+			Assert.True (l.RaiseListChangedEvents, "4");
+
+			Assert.False (ibl.IsSorted, "5");
+			Assert.Equal (ibl.SortDirection, ListSortDirection.Ascending);
+			Assert.True (ibl.SupportsChangeNotification, "7");
+			Assert.False (ibl.SupportsSearching, "8");
+			Assert.False (ibl.SupportsSorting, "9");
+			Assert.False (((IRaiseItemChangedEvents)l).RaisesItemChangedEvents, "10");
```

Comment:
Nit: instead of a number, can we use the name of the thing being tested? e.g.
```C#
Assert.True(l.AllowEdit, nameof(l.AllowEdit));
```
?
Same comment applies elsewhere.